dev-cpp/gtkmm/gtkmm-3.24.5.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{8..10} )
```

Comment:
I think there should be no Py 3.10 changes in this commit, unless it is a strict dependency.

Reply:
It's fine if it works and I think they needed to do it to allow Inkscape to get Python 3.10.

Reply:
![Screenshot_20211230_083726](https://user-images.githubusercontent.com/5751047/147731209-310176e3-9796-4fab-9aee-2226cf41610d.png)
python 3.10 depgraph shows *r+bdep* for inkscape 
source: gentoo python 3.10 qa report https://qa-reports.gentoo.org/output/gpyutils/39-to-310.svg

Reply:
... and yes Inkscape and many others work fine on my py310-only box since October.